src/com/company/IOManagerConsole.java

```diff
@@ -0,0 +1,129 @@
+package com.company;
+
+import java.util.Scanner;
+
+import static com.company.Main.MIN_BOARD_ID;
+import static com.company.Texts.*;
+
+public class IOManagerConsole implements IOManager {
+
+
+    @Override
+    public void showMessage(final String message) {
+        System.out.println(message);
+    }
+
+    @Override
+    public int getBoardSize() {
+        int result = 0;
+        Scanner INPUT = new Scanner(System.in);
+        try {
+            result = Integer.parseInt(INPUT.nextLine());
+        } catch (NumberFormatException error) {
+            showMessage(WRONG_FORMAT);
+        } catch (Exception error) {
+            error.printStackTrace();
+        }
+        return result;
+    }
+
+    @Override
+    public String getName() {
+        String result = "";
+        Scanner INPUT = new Scanner(System.in);
+        try {
+            result = INPUT.nextLine();  //next da tylko pierwsze słowo, nextLine odczyta wszystko przed enterem
+        } catch (Exception error) {
+            error.printStackTrace();
+        }
+        return result;
+    }
+
+    @Override
+    public Coordinates getCoordinates(final Board board) {
+
+        int min = MIN_BOARD_ID;
```

Comment:
`MIN_BOARD_ID` jest w mainie, fajnie by było gdyby IOManager był niezależny od czegokolwiek (żeby mógł być łatwo wymieniony)

Reply:
No to z kolei powstanie miejsce w którym trzeba będzie zmieniać to co można by było właśnie z maina wziąć, niestety wcześniej już dodałam w ten sam sposób kolejną zmienną i niestety nie mam pomysłu na tą niezależność. Można oczywiście zadeklarować nowe zmienne tymi samymi wartościami. Czy to jest dobry pomysł?

Reply:
przekazywać min i max w parametrach?

Reply:
no to sobie pogadałam sama ze sobą ... i zrobiłam... ;) 
ale
zastanawiam się czy lepiej przekazywać wielkość (max) pola przez zmienną czy za każdym razem (każdy wiersz i kolumna) pobierać ją z board ?

Reply:
Jeśli nie wykorzystujesz tego board do niczego innego to bez sensu wogóle go przekazywać, można go wtedy zastąpić maxem.